configs/complete-e2e.yaml

```
@@ -1,3 +1,13 @@
 tests:
   ginkgoLabelFilter: >
-    (E2E || Operators || TestHarness || ServiceDefinition || AppBuilds) && !Informing
\ No newline at end of file
+    (E2E || Operators || TestHarness || ServiceDefinition || AppBuilds) && !(Informing || MigratedHarness)
+  testHarnesses:
+  #    For migrated harnesses, remember to add "SkipHarness" label to existing suite in this repo, until they are removed from here.
```

```suggestion
  #    For migrated harnesses, remember to add "MigratedHarness" label to existing suite in this repo, until they are removed from here.
```

I think this should be `MigratedHarness` over `SkipHarness`?